src/target/renesas_rz.c

```diff
@@ -77,6 +77,7 @@
 #define RENESAS_OCRAM_BASE        0x20000000U
 #define RENESAS_OCRAM_MIRROR_BASE 0x60000000U
 #define RENESAS_OCRAM_SIZE        0x00200000U
+#define RENESAS_OCRAM_SIZE_A1L    0x002fffffU
```

Comment:
It would be unusual for the amount of RAM to be non-power-of-two - note this isn't the address of the last addressable byte, but how many there are, so a region defined as 0x20000000-0x202fffff has 0x00300000 bytes in it. Otherwise you'd be defining the region to be 0x20000000-0x202ffffe with this. Are you sure this isn't `0x00300000U`?

Reply:
ah good catch! Missed that. Will fix.